baggage/baggage.go

```diff
@@ -304,9 +304,33 @@ func parseMember(member string) (Member, error) {
 	if err != nil {
 		return newInvalidMember(), fmt.Errorf("%w: %v", errInvalidValue, err)
 	}
+
+	if !utf8.ValidString(value) {
+		// W3C baggage spec:
+		// https://github.com/w3c/baggage/blob/main/baggage/HTTP_HEADER_FORMAT.md?plain=1#L69
+		for _, is := range findInvalidUTF8Sequences(value) {
+			value = strings.ReplaceAll(value, is, "�")
+		}
+	}
+
 	return Member{key: key, value: value, properties: props, hasData: true}, nil
 }
 
+func findInvalidUTF8Sequences(input string) []string {
```

Comment:
Why not returning a string without invalid sequences instead?

Reply:
Do you mean do the `strings.ReplaceAll(.....)` here? I could do it.

Reply:
I mean do it so it replaces it "in flight" so that strings.ReplaceAll is not needed

Reply:
Sure, I will do it.

Reply:
@pellared I already did it, could you check it?

Reply:
Can you please update the benchstat result?

Reply:
@pellared done. Similar results than previous implementation. Let me know how you see the results.

Reply:
The problem is that the current benchmarks are not testing the proposed changes.

Reply:
@pellared I modified the algorithm to do just one allocation, I modified the benchmark test to add the use case and also I updated the benchmark results. Could you verify it again?

Reply:
LGTM. We just need to make sure to also fix parsing of properties as well.

Reply:
I did it.

Reply:
@pellared could you check it again? Thanks